hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/token/TokenCreateSpecs.java

```diff
@@ -999,7 +1013,11 @@ private HapiSpec prechecksWork() {
                                 .hasPrecheck(ACCOUNT_REPEATED_IN_ACCOUNT_AMOUNTS),
                         tokenAssociate(FIRST_USER, A_TOKEN),
                         cryptoTransfer(moving(0, A_TOKEN).between(TOKEN_TREASURY, FIRST_USER))
-                                .hasPrecheck(OK),
+                                // Either fail pre-check with EMPTY_TOKEN_TRANSFER_ACCOUNT_AMOUNTS (mono)
+                                // or post-consensus (modular). When we only have modular code left,
+                                // we can just check hasKnownStatus.
+                                .hasPrecheckFrom(OK, EMPTY_TOKEN_TRANSFER_ACCOUNT_AMOUNTS)
+                                .hasKnownStatus(EMPTY_TOKEN_TRANSFER_ACCOUNT_AMOUNTS),
```

Comment:
We should not be changing a passing transaction in mono-service to a failing one. Should preserve mono-service behavior